.nvmrc

```diff
@@ -0,0 +1 @@
+20
```

Comment:
What is this `20` here?

Reply:
Version 20 of nodejs. For [nvm](https://github.com/nvm-sh/nvm)